js/binance.js

```diff
@@ -2140,7 +2140,7 @@ module.exports = class binance extends Exchange {
             const fetchBalanceOptions = this.safeValue (options, 'fetchBalance', {});
             method = this.safeString (fetchBalanceOptions, 'method', 'dapiPrivateGetAccount');
             type = 'inverse';
-        } else if ((type === 'margin') || (marginMode === 'cross')) {
+        } else if ((type === 'margin') && (marginMode === 'cross')) {
```

Comment:
@pietrodn I understand your point, but we can't do this either because we want to default to `cross` if type == 'type' or `marginMode === 'cross'. To fix this we can change the order of the ifs and place the `isolated` above this one

Reply:
Hi @carlosmiei, I applied your suggestion, swapping the order of the `if`s and restoring the OR condition.